extensions/wikia/Insights/scripts/LoopTracking.js

```diff
@@ -0,0 +1,34 @@
+define(
+	'ext.wikia.Insights.LoopTracking',
+	[
+		'wikia.tracker'
+	],
+	function (tracker) {
+		'use strict';
+
+		var insightType;
+
+		function onLinkClick() {
+			// Track a click on a notification link
+			var trackingParams = {
+				trackingMethod: 'both',
+				category: 'insights',
+				action: tracker.ACTIONS.CLICK_LINK_TEXT,
+				label: insightType+'insight-list-item'
```

Comment:
I think we miss a dash '-' before the word insight. Can you also change the word 'insight' to 'insight*s*' to match the css classes etc.?